helix-rest/src/main/java/org/apache/helix/rest/server/resources/helix/PerInstanceAccessor.java

```diff
@@ -435,6 +435,11 @@ public Response updateInstance(@PathParam("clusterId") String clusterId,
       case setInstanceOperation:
          admin.setInstanceOperation(clusterId, instanceName, state);
          break;
+        case completeSwapIfPossible:
```

Comment:
nit: alignment?

Reply:
Also do we have canCompleteSwap?

Reply:
Added canCompleteSwap and fixed formatting.